XamarinCommunityToolkit/Views/CameraView/Android/MotionEventHelper.android.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Android.Views;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	class MotionEventHelper
+	{
+		VisualElement element;
+		bool isInViewCell;
+
+		public bool HandleMotionEvent(IViewParent parent, MotionEvent motionEvent)
+		{
+			if (isInViewCell || element == null || motionEvent == null || motionEvent.Action == MotionEventActions.Cancel)
+			{
+				return false;
+			}
+
+			var renderer = parent as VisualElementRenderer<Xamarin.Forms.View>;
+			if (renderer == null || ShouldPassThroughElement())
+			{
+				return false;
+			}
+
+			// Let the container know that we're "fake" handling this event
+			//renderer.NotifyFakeHandling();
+
+			return true;
+		}
+
+		public void UpdateElement(VisualElement element)
+		{
+			isInViewCell = false;
+			this.element = element;
+
+			if (this.element == null)
+			{
+				return;
+			}
+
+			// Determine whether this control is inside a ViewCell;
+			// we don't fake handle the events because ListView needs them for row selection
+			//isInViewCell = element.IsInViewCell();
+		}
+
+		bool ShouldPassThroughElement()
+		{
+			if (element is Layout layout)
+			{
+				if (!layout.InputTransparent)
+				{
+					// If the layout is not input transparent, then the event should not pass through it
+					return false;
+				}
+
+				if (layout.CascadeInputTransparent)
+				{
+					// This is a layout, and it's transparent, and all its children are transparent, then the event
+					// can just pass through 
+					return true;
+				}
+
+				if (Platform.GetRenderer(element) is VisualElementRenderer<Xamarin.Forms.View> renderer)
```

Comment:
comment out this test?

Reply:
I'll keep this, for now, I'll try to ping the original author and see if he remembers what those bits do.

Reply:
Let's timebox this in a way. If we don't hear from him in the next day we should assume we fix this ourselves. Then I'm interested to know; can we do that in a reasonable timeframe or will we merge it first and create an issue for it. Or, is it that crucial that we can't merge because it will be unusable?

Reply:
@bmacombe  @jfversluis I [sent a message](https://github.com/xamarin/Xamarin.Forms/pull/4084#issuecomment-688476068) to him yesterday, so I can wait at the end of the day tomorrow.
From my point of view, we can merge this PR and open issues for enhancements and fixes.

Reply:
removed